burnman/eos/cork.py

```diff
@@ -131,6 +131,11 @@ def validate_parameters(self, params):
         Check for existence and validity of the parameters
         """
 
+        if 'T_0' not in params:
+            params['T_0'] = 298.15
+        if 'P_0' not in params:
+            params['P_0'] = 0.
```

Comment:
P_0 is never used in the original expressions, so introducing a standard state pressure of 1.e5 would produce a (small) difference between burnman output and the original results. I favored this alternative...